### DISCUSSION TITLE
[Feature] Share albums across servers

### DISCUSSION THREAD

**Discussion by benniblot** [immich-app/immich#8900] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A feature to share / sync an album between servers would be nice. This would make it possible to have two friends host their own Immich server with private and shared photos, while being able to sync an album between them.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by zackpollard** [comment]
Hey! This is a duplicate of #916 which then links to #728 where we rejected this feature request in the past. This is something we have spoken about and something I personally want to see implemented in Immich. It's not a simple feature and probably requires us to be in a more "stable" position before it gets implemented, however it is something we are re-evaluating every so often and is on my personal to-do list for features to add to Immich.
